### PR TITLE
Frontier - Add Gravity Bridge DAI, ETH, USDC, USDT

### PR DESCRIPTION
**Pull request by jkilpatr**
This commit adds metadata for major Ethereum tokens brought to Osmosis
over Gravity Bridge. This will allow users of these pools to see the
token name and decimals.

### CODE REVIEW

**Review by ankurdotb — APPROVED** [review]

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -1390,7 +1390,128 @@
...
+      "name": "Tether USD",
+      "display": "USDT",
+      "symbol": "USDT",
+      "ibc": {
+        "source_channel": "channel-10",
+        "dst_channel": "channel-144",
+        "source_denom": "gravity0xdAC17F958D2ee523a2206206994597C13D831ec7"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/usdc.png",
```
Comment (ChristianBorst) [inline osmosis-1/osmosis-1.assetlist.json:1511]: Change to usdt.png/svg images

File: osmosis-1/osmosis-1.assetlist.json [hunk 2]
```diff
@@ -1390,7 +1390,128 @@
...
+      "name": "Dai Stablecoin",
+      "display": "DAI",
+      "symbol": "DAI",
+      "ibc": {
+        "source_channel": "channel-10",
+        "dst_channel": "channel-144",
+        "source_denom": "gravity0x6B175474E89094C44Da98b954EedeAC495271d0F"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/usdc.png",
```
Comment (ChristianBorst) [inline osmosis-1/osmosis-1.assetlist.json:1481]: Change to dai.png/svg images

File: osmosis-1/osmosis-1.assetlist.json [hunk 3]
```diff
@@ -1390,7 +1390,128 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/crbrus.png"
       },
-      "coingecko_id": "cerberus-2"
+      "coingecko_id": "cerberus-2",
+      "png": "https://cerberus-chain.s3.us-east-2.amazonaws.com/main/crbrus.png"
```
Comment (ChristianBorst) [inline osmosis-1/osmosis-1.assetlist.json:1394]: Remove this line

Comment (jkilpatr) [reply osmosis-1/osmosis-1.assetlist.json]: thanks for flagging resolved.

File: osmosis-1/osmosis-1.assetlist.json (hunk 2 above)
Comment (jkilpatr) [reply osmosis-1/osmosis-1.assetlist.json]: thanks for flagging resolved.

File: osmosis-1/osmosis-1.assetlist.json (hunk 1 above)
Comment (jkilpatr) [reply osmosis-1/osmosis-1.assetlist.json]: thanks for flagging resolved.

**Review by ankurdotb — APPROVED** [review]

**Review by IbrarMakaveli — CHANGES_REQUESTED** [review]
Can you switch your token PR on assetlist frontier : `osmosis-frontier.assetlist.json`
Thanks !

**Comment by jkilpatr** [comment]
closed in favor of #144

**Pull request closed without merging**